scipy/sparse/tests/test_base.py

```diff
@@ -4137,6 +4137,31 @@ def test_binop_explicit_zeros(self):
         assert (a + b).nnz == 2
         assert a.multiply(b).nnz == 1
 
+    def test_broadcast_to(self):
+        a = np.array([[1, 0, 2]])
+        b = np.array([[1], [0], [2]])
+        c = np.array([[1, 0, 2], [0, 3, 0]])
+        d = np.array([[7]])
+        e = np.array([[0]])
+        f = np.array([[0,0,0,0]])
+        res_a = csr_matrix(a).broadcast_to((2,3))
```

Comment:
Tests are failing because this method was renamed but the test wasn't updated to match.

Reply:
Fixed at 7cfdae829e38ef97b44922a03ca70822b5437801.